{"uid":"6ee5095c609fc762","name":"TC 17.01.25 Verify error on resetting My Password with empty email","fullName":"tests.test_forgot_your_password.TestForgotYourPasswordFunctionality#test_17_01_25_error_if_reset_my_password_with_empty_email","historyId":"3fc451d87c8653c0431c5bf955165f1b","time":{"start":1738978268071,"stop":1738978274103,"duration":6032},"description":"Check error message on attempt to reset password with empty email field","descriptionHtml":"<p>Check error message on attempt to reset password with empty email field</p>\n","status":"skipped","statusMessage":"XFAIL Flaky behavior on CI\n\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5636335b053a <unknown>\n#1 0x5636330abf00 <unknown>\n#2 0x5636330fbc0c <unknown>\n#3 0x5636330fbe31 <unknown>\n#4 0x563633141bd4 <unknown>\n#5 0x5636331205cd <unknown>\n#6 0x56363313ef84 <unknown>\n#7 0x563633120343 <unknown>\n#8 0x5636330ed78a <unknown>\n#9 0x5636330ee9de <unknown>\n#10 0x56363357a2cb <unknown>\n#11 0x56363357e242 <unknown>\n#12 0x5636335677ac <unknown>\n#13 0x56363357edf7 <unknown>\n#14 0x56363354bb2f <unknown>\n#15 0x56363359f1a8 <unknown>\n#16 0x56363359f370 <unknown>\n#17 0x5636335af3b6 <unknown>\n#18 0x7fdc22e9ca94 <unknown>\n#19 0x7fdc22f29c3c <unknown>","statusTrace":"self = <test_forgot_your_password.TestForgotYourPasswordFunctionality object at 0x7f864b1ea750>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0cd2e47eae3f7128a7815549d44b2bbc\")>\nforgot_psw_page = <pages.forgot_your_password_page.ForgotYourPasswordPage object at 0x7f86491f9c10>\n\n    @pytest.mark.xfail(reason=\"Flaky behavior on CI\")\n    @allure.title('TC 17.01.25 Verify error on resetting My Password with empty email')\n    def test_17_01_25_error_if_reset_my_password_with_empty_email(self, driver, forgot_psw_page):\n        \"\"\"Check error message on attempt to reset password with empty email field\"\"\"\n        # forgot_psw_page.fill_in_forgot_psw_captcha_field(captcha)\n        forgot_psw_page.click_reset_my_password_button()\n>       error_message = forgot_psw_page.get_error_email_input_is_required()\n\ntests/test_forgot_your_password.py:211: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/forgot_your_password_page.py:139: in get_error_email_input_is_required\n    return self.get_error_message(self.locators.RESET_MY_PASSWORD_EMAIL_ERROR)\npages/base_page.py:337: in get_error_message\n    error = wait(self.driver, timeout).until(EC.visibility_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"0cd2e47eae3f7128a7815549d44b2bbc\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f86491bdda0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5636335b053a <unknown>\nE       #1 0x5636330abf00 <unknown>\nE       #2 0x5636330fbc0c <unknown>\nE       #3 0x5636330fbe31 <unknown>\nE       #4 0x563633141bd4 <unknown>\nE       #5 0x5636331205cd <unknown>\nE       #6 0x56363313ef84 <unknown>\nE       #7 0x563633120343 <unknown>\nE       #8 0x5636330ed78a <unknown>\nE       #9 0x5636330ee9de <unknown>\nE       #10 0x56363357a2cb <unknown>\nE       #11 0x56363357e242 <unknown>\nE       #12 0x5636335677ac <unknown>\nE       #13 0x56363357edf7 <unknown>\nE       #14 0x56363354bb2f <unknown>\nE       #15 0x56363359f1a8 <unknown>\nE       #16 0x56363359f370 <unknown>\nE       #17 0x5636335af3b6 <unknown>\nE       #18 0x7fdc22e9ca94 <unknown>\nE       #19 0x7fdc22f29c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1738978266078,"stop":1738978267332,"duration":1254},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"forgot_psw_page","time":{"start":1738978267332,"stop":1738978268071,"duration":739},"status":"passed","steps":[{"name":"Open a browser","time":{"start":1738978267332,"stop":1738978268071,"duration":739},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Check error message on attempt to reset password with empty email field","status":"skipped","statusMessage":"XFAIL Flaky behavior on CI\n\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5636335b053a <unknown>\n#1 0x5636330abf00 <unknown>\n#2 0x5636330fbc0c <unknown>\n#3 0x5636330fbe31 <unknown>\n#4 0x563633141bd4 <unknown>\n#5 0x5636331205cd <unknown>\n#6 0x56363313ef84 <unknown>\n#7 0x563633120343 <unknown>\n#8 0x5636330ed78a <unknown>\n#9 0x5636330ee9de <unknown>\n#10 0x56363357a2cb <unknown>\n#11 0x56363357e242 <unknown>\n#12 0x5636335677ac <unknown>\n#13 0x56363357edf7 <unknown>\n#14 0x56363354bb2f <unknown>\n#15 0x56363359f1a8 <unknown>\n#16 0x56363359f370 <unknown>\n#17 0x5636335af3b6 <unknown>\n#18 0x7fdc22e9ca94 <unknown>\n#19 0x7fdc22f29c3c <unknown>","statusTrace":"self = <test_forgot_your_password.TestForgotYourPasswordFunctionality object at 0x7f864b1ea750>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0cd2e47eae3f7128a7815549d44b2bbc\")>\nforgot_psw_page = <pages.forgot_your_password_page.ForgotYourPasswordPage object at 0x7f86491f9c10>\n\n    @pytest.mark.xfail(reason=\"Flaky behavior on CI\")\n    @allure.title('TC 17.01.25 Verify error on resetting My Password with empty email')\n    def test_17_01_25_error_if_reset_my_password_with_empty_email(self, driver, forgot_psw_page):\n        \"\"\"Check error message on attempt to reset password with empty email field\"\"\"\n        # forgot_psw_page.fill_in_forgot_psw_captcha_field(captcha)\n        forgot_psw_page.click_reset_my_password_button()\n>       error_message = forgot_psw_page.get_error_email_input_is_required()\n\ntests/test_forgot_your_password.py:211: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/forgot_your_password_page.py:139: in get_error_email_input_is_required\n    return self.get_error_message(self.locators.RESET_MY_PASSWORD_EMAIL_ERROR)\npages/base_page.py:337: in get_error_message\n    error = wait(self.driver, timeout).until(EC.visibility_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"0cd2e47eae3f7128a7815549d44b2bbc\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f86491bdda0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5636335b053a <unknown>\nE       #1 0x5636330abf00 <unknown>\nE       #2 0x5636330fbc0c <unknown>\nE       #3 0x5636330fbe31 <unknown>\nE       #4 0x563633141bd4 <unknown>\nE       #5 0x5636331205cd <unknown>\nE       #6 0x56363313ef84 <unknown>\nE       #7 0x563633120343 <unknown>\nE       #8 0x5636330ed78a <unknown>\nE       #9 0x5636330ee9de <unknown>\nE       #10 0x56363357a2cb <unknown>\nE       #11 0x56363357e242 <unknown>\nE       #12 0x5636335677ac <unknown>\nE       #13 0x56363357edf7 <unknown>\nE       #14 0x56363354bb2f <unknown>\nE       #15 0x56363359f1a8 <unknown>\nE       #16 0x56363359f370 <unknown>\nE       #17 0x5636335af3b6 <unknown>\nE       #18 0x7fdc22e9ca94 <unknown>\nE       #19 0x7fdc22f29c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Click on Reset My Password button button","time":{"start":1738978268071,"stop":1738978268962,"duration":891},"status":"passed","steps":[{"name":"Check Reset My Password button is clickable","time":{"start":1738978268071,"stop":1738978268099,"duration":28},"status":"passed","steps":[{"name":"Find clickable elements","time":{"start":1738978268071,"stop":1738978268099,"duration":28},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"locator","value":"('css selector', 'button[class=\"action submit primary\"]')"},{"name":"timeout","value":"5"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Get error for email input is required ","time":{"start":1738978268962,"stop":1738978274102,"duration":5140},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5636335b053a <unknown>\n#1 0x5636330abf00 <unknown>\n#2 0x5636330fbc0c <unknown>\n#3 0x5636330fbe31 <unknown>\n#4 0x563633141bd4 <unknown>\n#5 0x5636331205cd <unknown>\n#6 0x56363313ef84 <unknown>\n#7 0x563633120343 <unknown>\n#8 0x5636330ed78a <unknown>\n#9 0x5636330ee9de <unknown>\n#10 0x56363357a2cb <unknown>\n#11 0x56363357e242 <unknown>\n#12 0x5636335677ac <unknown>\n#13 0x56363357edf7 <unknown>\n#14 0x56363354bb2f <unknown>\n#15 0x56363359f1a8 <unknown>\n#16 0x56363359f370 <unknown>\n#17 0x5636335af3b6 <unknown>\n#18 0x7fdc22e9ca94 <unknown>\n#19 0x7fdc22f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/forgot_your_password_page.py\", line 139, in get_error_email_input_is_required\n    return self.get_error_message(self.locators.RESET_MY_PASSWORD_EMAIL_ERROR)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 337, in get_error_message\n    error = wait(self.driver, timeout).until(EC.visibility_of_element_located(locator))\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Get error message","time":{"start":1738978268962,"stop":1738978274102,"duration":5140},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5636335b053a <unknown>\n#1 0x5636330abf00 <unknown>\n#2 0x5636330fbc0c <unknown>\n#3 0x5636330fbe31 <unknown>\n#4 0x563633141bd4 <unknown>\n#5 0x5636331205cd <unknown>\n#6 0x56363313ef84 <unknown>\n#7 0x563633120343 <unknown>\n#8 0x5636330ed78a <unknown>\n#9 0x5636330ee9de <unknown>\n#10 0x56363357a2cb <unknown>\n#11 0x56363357e242 <unknown>\n#12 0x5636335677ac <unknown>\n#13 0x56363357edf7 <unknown>\n#14 0x56363354bb2f <unknown>\n#15 0x56363359f1a8 <unknown>\n#16 0x56363359f370 <unknown>\n#17 0x5636335af3b6 <unknown>\n#18 0x7fdc22e9ca94 <unknown>\n#19 0x7fdc22f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 337, in get_error_message\n    error = wait(self.driver, timeout).until(EC.visibility_of_element_located(locator))\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[{"name":"locator","value":"('css selector', '#email_address-error')"},{"name":"timeout","value":"5"}],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"f97d9d65cab5e47a","name":"Screenshot","source":"f97d9d65cab5e47a.png","type":"image/png","size":62467},{"uid":"33605218d35b422c","name":"HTML source","source":"33605218d35b422c.html","type":"text/html","size":80034}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":5,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1738978274264,"stop":1738978274336,"duration":72},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"epic","value":"Forgot Your Password Functionality"},{"name":"tag","value":"@pytest.mark.xfail(reason='Flaky behavior on CI')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_forgot_your_password"},{"name":"subSuite","value":"TestForgotYourPasswordFunctionality"},{"name":"host","value":"fv-az1068-564"},{"name":"thread","value":"2852-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_forgot_your_password"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":2,"passed":5,"unknown":0,"total":7},"items":[{"uid":"b523300c6ac648c3","reportUrl":"https://verafes.github.io/luma_project/579//#testresult/b523300c6ac648c3","status":"skipped","statusDetails":"XFAIL Flaky behavior on CI\n\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5583d78db53a <unknown>\n#1 0x5583d73d6f00 <unknown>\n#2 0x5583d7426c0c <unknown>\n#3 0x5583d7426e31 <unknown>\n#4 0x5583d746cbd4 <unknown>\n#5 0x5583d744b5cd <unknown>\n#6 0x5583d7469f84 <unknown>\n#7 0x5583d744b343 <unknown>\n#8 0x5583d741878a <unknown>\n#9 0x5583d74199de <unknown>\n#10 0x5583d78a52cb <unknown>\n#11 0x5583d78a9242 <unknown>\n#12 0x5583d78927ac <unknown>\n#13 0x5583d78a9df7 <unknown>\n#14 0x5583d7876b2f <unknown>\n#15 0x5583d78ca1a8 <unknown>\n#16 0x5583d78ca370 <unknown>\n#17 0x5583d78da3b6 <unknown>\n#18 0x7f52b269ca94 <unknown>\n#19 0x7f52b2729c3c <unknown>","time":{"start":1738892519986,"stop":1738892526080,"duration":6094}},{"uid":"75efd10b350029ce","reportUrl":"https://verafes.github.io/luma_project/578//#testresult/75efd10b350029ce","status":"passed","statusDetails":"XPASS Flaky behavior on CI","time":{"start":1738806074442,"stop":1738806074561,"duration":119}},{"uid":"77dfc3fcfe4292a0","reportUrl":"https://verafes.github.io/luma_project/577//#testresult/77dfc3fcfe4292a0","status":"passed","statusDetails":"XPASS Flaky behavior on CI","time":{"start":1738719696881,"stop":1738719696991,"duration":110}},{"uid":"3cf604cc0eed039a","reportUrl":"https://verafes.github.io/luma_project/576//#testresult/3cf604cc0eed039a","status":"passed","statusDetails":"XPASS Flaky behavior on CI","time":{"start":1738633189169,"stop":1738633189300,"duration":131}},{"uid":"b02d07a034e1657c","reportUrl":"https://verafes.github.io/luma_project/575//#testresult/b02d07a034e1657c","status":"passed","statusDetails":"XPASS Flaky behavior on CI","time":{"start":1738546855672,"stop":1738546855794,"duration":122}},{"uid":"bf7f67470c66ab29","reportUrl":"https://verafes.github.io/luma_project/574//#testresult/bf7f67470c66ab29","status":"passed","statusDetails":"XPASS Flaky behavior on CI","time":{"start":1738460597661,"stop":1738460597911,"duration":250}}]},"tags":["@pytest.mark.xfail(reason='Flaky behavior on CI')"]},"source":"6ee5095c609fc762.json","parameterValues":[]}